org.opent2t.sample.thermostat.superpopular/com.wink.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,65 @@
+/*jshint esversion: 6 */
+var opent2t = require('opent2t');
+var ava = require('ava');
+var config = require('./testconfig');
+var delay = require('delay')
+
+// This boilerplate code is copied from transpiled typescript AVA test code, which enables usage of 'yield' keyword. 
+var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, P, generator) {

```

Comment:
We can have all those benefits while using normal .then() callbacks instead of this __awaiter which is messy and unfamiliar to people. AVA doesn't require TypeScript or await.

---
In reply to: [75208579](https://github.com/openT2T/translators/pull/53#discussion_r75208579) [](ancestors = 75208579,75198452)